azure-maven-plugin-lib/src/main/java/com/microsoft/azure/toolkit/maven/common/messager/MavenAzureMessager.java

```diff
@@ -5,37 +5,30 @@
 
 package com.microsoft.azure.toolkit.maven.common.messager;
 
+import com.microsoft.azure.toolkit.lib.common.messager.IAzureMessage;
 import com.microsoft.azure.toolkit.lib.common.messager.IAzureMessager;
 import com.microsoft.azure.toolkit.lib.common.utils.TextUtils;
 import lombok.extern.slf4j.Slf4j;
 
-import javax.annotation.Nonnull;
-
 @Slf4j
 public class MavenAzureMessager implements IAzureMessager {
     @Override
-    public void success(@Nonnull String message, @Nonnull String title) {
-        log.info(message);
-    }
-
-    @Override
-    public void info(@Nonnull String message, @Nonnull String title) {
-        log.info(message);
-    }
-
-    @Override
-    public void warning(@Nonnull String message, @Nonnull String title) {
-        log.warn(message);
-    }
-
-    @Override
-    public void error(@Nonnull String message, @Nonnull String title) {
-        log.error(message);
-    }
-
-    @Override
-    public void error(@Nonnull Throwable throwable, @Nonnull String title) {
-        log.error(title, throwable);
+    public boolean show(IAzureMessage message) {
```

Comment:
will it open any dialog or window ?
or just print message into console / file ?

Reply:
it won't open any dialog/window in maven.